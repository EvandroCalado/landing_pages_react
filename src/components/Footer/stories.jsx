import Footer from '.';

const FooterStories = {
  title: 'Footer',
  component: Footer,
  args: {
    footerHtml: `<p><a href="https://github.com/EvandroCalado">Feito com ðŸ’• por Evandro Calado</a></p>`,
  },
};

export const Template = (args) => {
  return (
    <div>
      <Footer {...args} />
    </div>
  );
};

export default FooterStories;
