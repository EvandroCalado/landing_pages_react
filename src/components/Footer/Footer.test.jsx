import Footer from '.';
import { renderTheme } from '../../styles/render.theme';

describe('<Footer />', () => {
  it('should render', () => {
    const { container } = renderTheme(<Footer>Children</Footer>);
    expect(container).toMatchSnapshot();
  });
});
