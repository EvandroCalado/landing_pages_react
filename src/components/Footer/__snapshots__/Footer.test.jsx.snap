// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render 1`] = `
.c3 {
  font-size: 2.4rem;
}

.c0 {
  text-align: center;
  border-top: 1px solid #ddd;
}

.c0 a {
  color: inherit;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c0 .c2 {
  font-size: 1.6rem;
}

.c1 {
  width: 100%;
  max-width: 120rem;
  margin: 0 auto;
  padding: 3.2rem;
}

<div>
  <footer
    class="c0"
  >
    <div
      class="c1"
    >
      <div
        class="c2 c3"
      >
        <h1>
          Ol√°
        </h1>
      </div>
    </div>
  </footer>
</div>
`;
